packages/fxa-payments-server/src/lib/amplitude.ts

```diff
@@ -31,6 +31,7 @@ const eventTypeNames = {
 
 type GlobalEventProperties = {
   uid?: string;
+  subscribed_plan_ids?: string;
```

Comment:
Nit: This isn't a global property, since it's only present on some `fxa_pay_setup - *` and `fxa_pay_subscription_change - *` events -- move to `EventProperties` below?

Reply:
done